pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -216,3 +220,34 @@ func findContainerIDByName(status *v1.PodStatus, name string) (string, error) {
 	}
 	return "", fmt.Errorf("[cpumanager] unable to find ID for container with name %v in pod status (it may not be running)", name)
 }
+
+// Admit rejects a pod if its not safe to admit for node stability.
```

Comment:
Actually I just added `lifecycle.PodAdmitHandler` to the `Manager` interface and I already have a type assertion for that.  Comment added also.